v03_pipeline/var/test/reference_data/test_combined_mito_1.ht/README.txt

```diff
@@ -1,3 +1,3 @@
 This folder comprises a Hail (www.hail.is) native Table or MatrixTable.
-  Written with version 0.2.133-4c60fddb171a
-  Created at 2024/11/02 15:10:48
\ No newline at end of file
+  Written with version 0.2.130-bea04d9c79b5
+  Created at 2024/11/06 13:26:14
```

Comment:
I altered this table so that it only has the mitimpact annotations with the old values, to be used in updated_reference_dataset_collection_test